converters/toZigbee.js

```diff
@@ -391,6 +391,47 @@ const converters = {
             } else if (value.hasOwnProperty('hex') || (typeof value === 'string' && value.startsWith('#'))) {
                 const xy = utils.hexToXY(typeof value === 'string' && value.startsWith('#') ? value : value.hex);
                 value = {x: xy.x, y: xy.y};
+            } else if (value.hasOwnProperty('h') && value.hasOwnProperty('s') && value.hasOwnProperty('l')) {
+                const hsb = utils.hslToHsb(value.h, value.s, value.l);
+                newState.color = {h: hsb.h, s: hsb.s, b: hsb.b};
+                value.hue = hsb.h % 360 * (65535 / 360);
+                value.saturation = hsb.s * (2.54);
+                value.brightness = hsb.b * (2.54);
+                cmd = 'enhancedMoveToHueAndSaturationAndBrightness';
+            } else if (value.hasOwnProperty('hsl')) {
+                const hsl = value.hsl.split(',').map((i) => parseInt(i));
+                const hsb = utils.hslToHsb(hsl.h, hsl.s, hsl.l);
+                newState.color = {h: hsb.h, s: hsb.s, b: hsb.b};
```

Comment:
Should this not return h, s, and l instead of h, s, and b for newState.color ?

Reply:
good catch, fixed it